randomActivityGen.py

```diff
@@ -246,6 +248,46 @@ def verify_stats(stats: ET.ElementTree):
         ET.SubElement(work_hours, "closing", {"hour": "63000", "proportion": "60"})  # 60% at 17.30
 
 
+def setup_bus_stops(net: sumolib.net.Net, stats: ET.ElementTree, min_distance, k):
+    edges = net.getEdges()
+
+    city = stats.getroot()
+    bus_stations = city.find("busStations")
+    seed_bus_stops = []
+    if bus_stations is None:
+        bus_stations = ET.SubElement(city, "busStations")
+    else:
+        for station in bus_stations.findall("busStation"):
+            assert "edge" in station.attrib, "BusStation isn't placed on an edge"
+            edge_id = station.attrib["edge"]
+            assert "pos" in station.attrib, "BusStation doesn't have a position along the edge"
+            along = float(station.attrib["pos"])
+
+            for edge in edges:
+                if edge.getID() == edge_id:
```

Comment:
Use `net.getEdge(edge_id)` instead of iterating through them

Reply:
fixed in 434f43f